sdcm/utils/nemesis.py

```diff
@@ -122,8 +121,7 @@ def create_job_files_from_template(self) -> list[Path]:
                 {
                     "nemesis_longevity_config": config_name,
                     "backend": self.backend,
-                    "region": self.BACKEND_TO_REGION.get(self.backend, "eu-west-1"),
-                    "additional_params": additional_params,
+                    "region": self.BACKEND_TO_REGION.get(self.backend, "eu-west-1")
```

Comment:
This is redundant line change. Why removing comma?

Reply:
fixed

Reply:
@soyacz right, it was not my change. But I will add the comma here